mautrix_telegram/commands/handler.py

```diff
@@ -14,17 +14,29 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
-from typing import Awaitable, Callable, Dict, List, NamedTuple, Optional
-import traceback
+"""This module contains classes handling commands issued by Matrix users."""
+from typing import (
+    Any,
+    Awaitable,
+    Callable,
+    Coroutine,
+    Dict,
+    List,
+    NamedTuple,
+    Optional,
+    Union,
+    NewType,
+)
 import logging
+import traceback
 
 import commonmark
 
 from telethon.errors import FloodWaitError
 
-from ..types import MatrixRoomID, MatrixEventID
-from ..util import format_duration
-from .. import user as u, context as c
+from mautrix_telegram.types import MatrixRoomID, MatrixEventID
+from mautrix_telegram.util import format_duration
+from mautrix_telegram import user as u, context as c
```

Comment:
Do these have to be changed for the tests to work? I currently use relative imports everywhere.

Reply:
No, they don't affect the tests. That was just a habit that got through, so I reverted the change.